packages/card/src/Card.ts

```diff
@@ -189,12 +189,12 @@ export class Card extends FocusVisiblePolyfillMixin(SpectrumElement) {
             ${this.renderImage()}
             <div class="body">
                 <div class="header">
-                    ${this.renderTitle}
+                    ${this.renderHeading}
                     ${this.variant === 'gallery'
                         ? html`
                               <div class="subtitle">
```

Comment:
We need to alter this `.subtitle` selector as well?

Reply:
The classes are referenced in the generated spectrum css (I had to back out some more aggressive changes here). I could probably update the unit tests though since there are tests on the `title` class being present, but I think not the `subtitle`.

Reply:
Seems related to this change: https://github.com/adobe/spectrum-web-components/pull/945/files#diff-f19f38486d97656b69421377bd02c585R34

Reply:
These diffs outline that: https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5bf34b122c07f700013ff527/5f84b15c3b814704c2f4ad7b-0-build/artifacts/test/visual/screenshots-current/card--actions__dark__large__ltr-diff.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201012T195001Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20201012%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=4c185963921609b19f71180bce974b25730cca3f573a66f766049065bcd95100

Reply:
Made a comment on the config file. The diff link you shared expired, but I do see some diffences where the subheading text renders differently if that's what you're talking about (for those, the text didn't change, so I'd expect it to be the same).

Reply:
![image](https://user-images.githubusercontent.com/1156657/95791434-5df3c900-0caf-11eb-8479-ffd56ecb507e.png)
For reference.